### DISCUSSION TITLE
PROPOSAL: Reducing the quorum threshold

### DISCUSSION THREAD

**Discussion by Terra854** [crypto-org-chain/chain-main#697] [Ideas]
Currently, a proposal is required to have a minimum of 33.4% of voting power in order to reach quorum. Without reaching that minimum threshold, the entire deposit is burned.

That will disincentivise stakeholders from depositing on a beneficial proposal as it will carry the risk of losing the entire amount deposited should the proposal fail to reach quorum. What's more, looking back at the proposal history, there is a significant amount of non-CdC validators that have historically did not vote on any of the proposals. After calculating, that will leave us with only an estimated 26% of voting power that have voted in at least 1 proposal, making it impossible for proposals to hit the quorum threshold without CdC intervention.

I would suggest reducing the quorum threshold to around 15% or 20% so that we will no longer need to rely on CdC in order to reach  a quorum and more people can deposit on proposals without have to worry too much about whether there is enough people voted to be able to get the deposit back.

**Comment by samuel-tay** [comment]
I agree with 20%. This will bring decentralization as CDC can simply not vote to imply abstain

**Comment by eeldivad** [comment]
Reducing quorum is good if there are no other options.  So far no one has lost their deposits and CDC has voted every time to ensure that.  CDC should put more effort into reducing their voting power and spreading it out to smaller validators.  All they have to do is unstake a portion of their self delegation or raise their commissions to encourage users to redelegate.  The purpose of quorum is for security and encourage more to vote.  If we can't achieve at least one third of the votes then we have a different problem to address, which is how do we encourage more validators to vote.  One option is we should penalize validators that do not take part in governance.  So if they miss 2 proposals in a row then slash 1% or something.

**Comment by salvatore-galaxystaking** [comment]
In principle, we would agree to this plan, but we think that the problem is a different one.
As already pointed out by @eeldivad if we cannot get to vote 1/3 of the VP then we have other problems.
We would set more priority on a redelegation from top to bottom of the VP and some initiative on the so called "0% war".
With a relatively good distribution of voting power, you might only need 40-50 voting validators to reach  >33%. and that should be possible in every vote.

**Reply by tomtau** [reply]
It looks like the root cause of the current situation is that a few of the big validators don't participate in the network governance.
Instead of reducing the quorum, it'd be good to brainstorm other options that could address the root cause. Just a few ideas:

1. Getting those validators involved to understand why they didn't participate in them: Was it because they were indifferent to the proposals? (In which case, could they just vote Abstain on them?) Or was it because they missed them? And what would help them not to miss proposals in the future (e.g. a Telegram notification bot that would monitor the gov proposals)?

2. Unlike on Cosmos Hub where it was due to validators operated by big exchanges (which self-delegated funds in their custody), the assumption is that the non-participating validators here have their VP mainly from external delegations. So one alternative will be to encourage delegators to participate in the network governance. For example, wallets could have active proposal notifications that would keep prompting delegators to vote until they do so.

**Reply by reefValidator** [reply]
i have to agree with you that the non-participation of some big ones is a problem. but the main problem is and remains the poor distribution of voting power.

to 1.) i think that every validator should have own interest in the network.
i see it like my nephew's godparenthood. he can't get in touch with me, so I have to be the active one. if this interest is available, you cannot "overlook" any proposal, due to the many discussions (for example discord).

to 2.) I would suggest to update the apps (desktop, ios, android) and integrate the proposal voting for delegators.

**Reply by samuel-tay** [reply]
I'm of the opinion that the validators have the responsibility to vote and participate in governance. The responsibility does not fallback onto the delegator just because the validator is lazy.

Large validators should be voting even if it were an abstain vote. If they can't be bothered to even vote for that, then we will need to reduce the quorum.
